apps/documentation/next.config.mjs

```diff
@@ -1,6 +1,55 @@
+import { createVanillaExtractPlugin } from '@vanilla-extract/next-plugin';
+import { PHASE_DEVELOPMENT_SERVER } from 'next/constants.js';
+
+const withVanillaExtract = createVanillaExtractPlugin({ ssr: true, identifiers: ({ hash }) => `vfui_${hash}` });
+
+const plugins = [withVanillaExtract];
+
+const phaseConfig = (phase, { defaultConfig }) => {
+  if (phase === PHASE_DEVELOPMENT_SERVER) {
+    return defaultConfig;
+  }
+
+  return {
+    typescript: {
+      tsconfigPath: 'tsconfig.build.json',
+    },
+  };
+};
+
+const pluginConfig = plugins.reduce((acc, plugin) => plugin(acc), phaseConfig);
+
 /** @type {import('next').NextConfig} */
+
 const nextConfig = {
-  reactStrictMode: true,
+  ...pluginConfig,
+  distDir: 'build',
+  webpack: (webpackConfig, context) => {
+    const baseConfig = pluginConfig.webpack(webpackConfig, context);
+
+    return {
+      ...baseConfig,
+
+      resolve: {
+        ...baseConfig?.resolve,
+
+        alias: {
+          ...baseConfig?.resolve?.alias,
+
+          '@voiceflow/ui': '@voiceflow/ui',
```

Comment:
are you sure this is necessary?  
I see all of these are the same, so they're not really 'aliases'.. ?

Reply:
removed